components/script/dom/attr.rs

```diff
@@ -244,6 +247,24 @@ impl<'a> AttrHelpers<'a> for JSRef<'a, Attr> {
         &self.extended_deref().local_name
     }
 
+    fn set_owner(self, owner: Option<JSRef<Element>>) {
```

Comment:
I suggest you leave a doc comment on this stating that it should be used _after_ having moved the attr